src/QsCompiler/LlvmBindings/Metadata/MDString.cs

```diff
@@ -0,0 +1,29 @@
+// -----------------------------------------------------------------------
+// <copyright file="MDString.cs" company="Ubiquity.NET Contributors">
+// Copyright (c) Ubiquity.NET Contributors. All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+using LLVMSharp.Interop;
+
+namespace Ubiquity.NET.Llvm
+{
+    /// <summary>Stores a string in Metadata</summary>
+    public class MDString
+        : LlvmMetadata
+    {
+        /// <summary>Gets the string from the metadata node</summary>
+        /// <returns>String this node wraps</returns>
+        public override string ToString( )
+        {
+            var context = ContextCache.Single();
```

Comment:
It might be worth reproducing the explanatory comment here too.

Reply:
https://github.com/microsoft/qsharp-compiler/pull/1026/commits/6f0de6f347cf1b872daa29dc25d9e466d645288f